setup/launch.py

```
@@ -227,24 +228,24 @@ def check_python_version():
             f"to {PYTHON_MAX_VERSION[0]}.{PYTHON_MAX_VERSION[1]}."
         )
->>>>>>> a7da61cf1f697de3c8c81f536bf579d36d88e613
+
         sys.exit(1)
     logger.info(f"Python version {platform.python_version()} is compatible.")
 
 
 # --- Environment Validation ---
-<<<<<<< HEAD
+
 def check_for_merge_conflicts() -> bool:
```

Flake8 (7.3.0)

[error] redefinition of unused 'check_for_merge_conflicts' from line 33

(F811)